<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Makao</class>
 <widget class="QDialog" name="Makao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>1921</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="oppLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>570</y>
     <width>1921</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="playerCardsLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>1921</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="tableCardsLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnGivePlayerCard">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>800</y>
     <width>951</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj karte</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>1720</x>
     <y>860</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRestart">
   <property name="geometry">
    <rect>
     <x>1720</x>
     <y>920</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>1720</x>
     <y>980</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyj≈õcie</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblState">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Stan gry</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
